.github/workflows/docs-pr-netlify.yaml

```diff
@@ -0,0 +1,34 @@
+name: Deploy documentation PR preview
+
+on:
+  workflow_run:
+    workflows: [ "Prepare documentation PR preview" ]
+    types:
+      - completed
+
+jobs:
+  netlify:
+    if: github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.event == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      # There's a 'download artifact' action, but it hasn't been updated for the workflow_run action
+      # (https://github.com/actions/download-artifact/issues/60) so instead we get this mess:
+      - name: 📥 Download artifact
+        uses: dawidd6/action-download-artifact@af92a8455a59214b7b932932f2662fdefbd78126 # v2.15.0
```

Comment:
Why are we using commit refs instead of tags?

Reply:
Tags can be replaced, breaking a hash is far more difficult

Reply:
Is this a thing we're afraid can realistically happen? I'm unconvinced maintainers of the two third-party workflows we use would ever do this, and even then it doesn't seem very difficult to fix to me.

Reply:
FTR it is copied from matrix-spec, so it'd be good to be org-consistent: https://github.com/matrix-org/matrix-spec/blob/main/.github/workflows/netlify.yaml#L37-L42